packages/components/modal/src/drawer-content.tsx

```diff
@@ -46,7 +48,7 @@ export const DrawerContent = motionForwardRef<DrawerContentProps, "div">(
     },
     ref,
   ) => {
-    const { describedbyId, duration, isOpen, labelledbyId, onClose } =
+    let { describedbyId, duration, isOpen, labelledbyId, open, onClose } =
```

Comment:
Why did you change from const to let? Is there a reason for that?

Reply:
Good catch! I have changed it back